CMakeLists.txt

```diff
@@ -109,8 +111,22 @@ list(APPEND CMAKE_PREFIX_PATH /opt/rocm /opt/rocm/llvm /opt/rocm/hip /opt/rocm/h
 option(ENABLE_HIP_WORKAROUNDS Off)
 set(MIOPEN_INSTALL_CXX_HEADERS Off CACHE BOOL "Install MIOpen's C++ header interface")
 
-option(MIOPEN_USE_COMGR "When possible, use comgr to build kernels instead of offline tools" Off)
-option(MIOPEN_USE_HIP_KERNELS "Use HIP kernels." On)
+
+# Embedded Build Configuration
+option(MIOPEN_EMBED_BUILD "Build with the set of embed flags." Off)
+option(MIOPEN_USE_COMGR "Use comgr to build kernels instead of offline tools" ${MIOPEN_EMBED_BUILD})
+option(MIOPEN_DISABLE_USERDB "Disable user database access" ${MIOPEN_EMBED_BUILD})
+
+# MIOPEN_USE_HIP_KERNELS is a Workaround for COMgr issues
+if(MIOPEN_EMBED_BUILD)
+    option(BUILD_SHARED_LIBS "Build as a shared library" Off)
```

Comment:
This actually wont set it to `Off` because it was already set on line 43.

Reply:
Ooff, missed that. Fixed.